### DISCUSSION TITLE
Rich Text

### DISCUSSION THREAD

**Discussion by krenarxh** [tasteui/tasteui#547] [Ideas]
It would be amazing if you can integrate in the text area component one of the rich texts packages

**Comment by devajmeireles** [comment]
Hey, @krenarxh ! We're preparing a new component, a text editor linked to this feature request, https://github.com/tallstackui/tallstackui/issues/433. Do you believe it will be enough for us in the TallStackUI?

**Reply by krenarxh** [reply]
Yes I think so, for this feature , I had a project that needed to give the admin the ability to edit the email template, and was looking for a google doc type of text editor, so the admin would not need html css knowledge to create a good looking email, I integrated the tinyMce editor right now after looking your doc for something similar, and thought to write you guys to implement something like that, I really love your UI keep up the amazing work. Thank you for making our job easier.
